docs/source/resources/common/exposing.md

```diff
@@ -237,16 +394,41 @@ spec:
       nodes:
         type: ServiceClusterIP
 ```
+:::
+:::{tab-item} ScyllaDBCluster
+```{include} ../../.internal/scylladbcluster-cluster-ip-exposure-caution.md
+```
+```yaml
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: ScyllaDBCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: ServiceClusterIP
+```
+:::
+::::
+
 
 In this scenario, we assume that the Pod IP subnet is routable within a VPC. 
 Clients within the VPC network can communicate directly with ScyllaCluster nodes using PodIPs.
 Nodes communicate with each other exclusively within the same Kubernetes cluster.
 
-![PodIPs](./exposing-images/podips.svg)
+![PodIPs](exposing-images/podips.svg)
 
 ### Multi VPC
 
-ScyllaCluster definition:
+::::{tab-set}
+:::{tab-item} ScyllaCluster
+
+Both ScyllaDB datacenters use the same `exposeOptions`, nodes broadcast their Pod IP addresses, enabling them to establish connections with one another.
+****Check other documentation pages to know how to connect two ScyllaClusters into one logical cluster.
```

Comment:
I think this doesn't render as intended (or I don't understand the purpose of the asterisks)
![image](https://github.com/user-attachments/assets/3ab52f92-db77-4faa-b462-45456a80016f)

Reply:
done